### DISCUSSION TITLE
v4 i18n.js

### DISCUSSION THREAD

**Discussion by kkmuffme** [fnando/i18n-js#666] [Q&A]
We were using i18n JS in vanilla JS via the provided i18n.js
This isn't available anymore in v4. Is there a way to use i18n-js via vanilla JS?

**Comment by andrewm-mitchells** [comment]
I'm having the same issue and would love to continue using the plugin in vanilla js. Our app is not using Webpacker to fully support ES5/ES6 stuff and there's no JS file in the gem to import.

**Comment by fnando** [comment]
Sorry for not answering this before, but [i18n](https://github.com/fnando/i18n) does export browser compiled files. It's stored under `dist/browser/index.js`. You can see it in action here: https://codepen.io/fnando/pen/wvXZRVe

The gem doesn't bundle JavaScript files anymore, so you need to install this out of npm. Given that i18n-js became used outside Ruby devs, I broke both packages apart; the rubygem i18n-js is just a CLI, while the i18n-js npm package contains everything related to JavaScript/TypeScript.
